index.hbs

```diff
@@ -76,11 +76,85 @@ into the {body} of the default.hbs template --}}
             {{/if}}
           {{/get}}
           {{!-- The tag below iterates over all the posts --}}
-          {{> "loop"}}
-        </div>
-      </div>
+          {{!-- {{> "loop"}} 
+        </div> 
+        </div> --}}
+
+        <div class="l-wrapper in-post aos-init aos-animate" data-aos="fade-up" data-aos-delay="300">
+            <div class="l-post-content">
+              <div class="pos-relative js-post-content">
+                <h1 class="ieee-blue">Who We Are</h1>
+                <h3 class="ieee-blue">Student Leaders</h2>
+                <div class="paragraph-div">
+                  <p class="paragraph">
+                    The University of California, Berkeley IEEE Student Branch was established in 1912 and has been one of the most active engineering student organizations on campus ever since.
+                  </p>
+                </div>
+
+                <h3 class="ieee-blue">What We Do</h1>
+                <div class="paragraph-div">
+                  <p class="paragraph">
+                    We aim to promote professional development, encourage academic growth, and organize social activities within the greater engineering community. <br />
+                    Some of the activities we host:
+                    <div class="row">
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Technical Workshops</li>
+                          <li>Resume and Interview Workshops</li>
+                          <li>Startup Fair</li>
+                          <li>Dinner with Corporate Sponsors</li>
+                        </ul>
+                      </div>
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Company Info-sessions</li>
+                          <li>Mentorship Program</li>
+                          <li>Two Decals (Hope, Micromouse)</li>
+                          <li>Socials (Dinner, bowling, ice-skating, etc.)</li>
+                        </ul>
+                      </div>
+                    </div>
+                  </p>
+                </div>  {{!-- End of paragraph-div --}}
+
+
+                <div class="stats-div text-center div-spacing">
```

Comment:
Looking back at the original design, the circle design is hard to pull off while looking nice, and I think a far more effective design choice that other websites use a lot is keeping the proportions of the original markup, but removing the circle and coloring the number and the subtitle text differently. I made a mockup of what that might look like: ![mockup](https://i.fluffy.cc/WlG8s72r2JpccKj8q1Q0Hqv5LvCJQ7Z1.png)

Reply:
Woah, that font looks really nice. Could you link the font used here?